Vehicle page changes

abhishekshree opened the pull request:
- Updated the file structure for vehicle pages
- Changed the UI as planned (Accordions and tabs added along with model-viewer)
- data.js datastore created.
For #30 @shubhamkorde

shubhamkorde reviewed (commented):
Make the following changes:
- The alignment of the heading on Anahita and Varun pages is not centered. It can be fixed by the textAlign center style on the div containing the heading and the subheading 
- Remove the shadow from the cad model container or make it more subtle. Something like 5px 5px 10px rgba(black, 0.2) should be fine. Adding can also be an option

Rest all is good. Good to be merged.

abhishekshree commented on src/views/Vehicles/AnahitaPage/components/SubsystemSpecs/subs.css:
```diff
@@ -1,22 +1,38 @@
-  padding-top: 20px;
-  padding-bottom: 20px;
+.spec-card:hover {
+  transition: none !important;
+  animation: none !important;
+  transform: none !important;
```
These tags are for removing the default card animation.

abhishekshree commented:
I have updated the PR @shubhamkorde 
The changes are:
- Added correct Varun Data and images
- Removed Shadows from the cad models
- Corrected CSS classnames.

abhishekshree commented on src/views/Vehicles/VarunPage/data.js:
```diff
@@ -0,0 +1,100 @@
+            "spec": "Ordroid X2 processor with Samsung Exynyos 4412 chip and 2GB RAM"
+        },
+        {
+            "name": "Something",
```
I didn't know what to title this...
